nucliadb/nucliadb/search/api/v1/find.py

```diff
@@ -121,6 +111,9 @@ async def find_knowledgebox(
     x_nucliadb_user: str = Header(""),
     x_forwarded_for: str = Header(""),
 ) -> Union[KnowledgeboxFindResults, HTTPClientError]:
+    if SearchOptions.DOCUMENT in features:
+        return HTTPClientError(status_code=422, detail="fulltext search not supported")
```

Comment:
NIT IDEA: to avoid duplication, you can try-except `FindRequest` creation and on validation error, raise a 422

Reply:
yes! good idea, I like it. Then we have just one validation point!